t/plugin/server-info.t

```diff
@@ -133,3 +133,51 @@ integral
 [error]
 --- error_log
 timer created to report server info, interval: 60
+
+
+
+=== TEST 3: get server_info from plugin control API
+--- yaml_config
+apisix:
+    id: 123456
+plugins:
+    - server-info
+--- config
+location /t {
+    content_by_lua_block {
+        local json_decode = require("apisix.core").json.decode
```

Comment:
https://github.com/api7/test-toolkit

we can use this `json` to print the sorted data. we can do this in a new PR